rxandroidble/src/main/java/com/polidea/rxandroidble/internal/connection/RxBleGattCallback.java

```diff
@@ -12,8 +12,8 @@
 import com.polidea.rxandroidble.exceptions.BleGattException;
 import com.polidea.rxandroidble.exceptions.BleGattOperationType;
 import com.polidea.rxandroidble.internal.RxBleLog;
-
```

Comment:
From my experience any added / deleted line that is not directly connected to the change only obfuscates the code. Minimal changes are the best changes as it is easier to spot mistakes.